x-pack/metricbeat/module/benchmark/info/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- name: info
+  type: group
+  release: beta
+  description: >
+    info
+  fields:
+    - name: counter
+      type: keyword
```

Comment:
Should this be a numeric type?

Reply:
See previous comment on why "numeric" data shouldn't always have a numeric type in elasticsearch.